src/js/common/schemaform/formState.js

```diff
@@ -6,6 +6,12 @@ function isHiddenField(schema) {
   return !!schema['ui:collapsed'] || !!schema['ui:hidden'];
 }
 
+function get(path, data) {
```

Comment:
It turns out that lodash-fp's `get` and `findLast` are a lot slower on low end devices than I expected. Removing them here shaves ~75-100ms off the updates that happen on every data change.

It's likely there are still more improvements related to that, but this change gets a good enough improvement for now without the risk of changing lots of instances.